src/main/resources/theme/theme_rds.scss

```diff
@@ -11,5 +11,7 @@ $main-logo: "img/logo-rds.svg";
 $primefaces-main: "img/primefaces-main-rds.png";
 $dataverse-icon: "img/dataverse-icon-rds.svg";
 $ajax_spinner_filter: hue-rotate(170deg) saturate(2);
+$dataverse-icon-mono: "\f200";
+$main-logo-mono: "\f203";
```

Comment:
we need also extra file: img/primefaces-main-yellow.png

Reply:
Fixed, I've also added `primefaces-main-white.png` so that this repo is ready for other high contrast themes, in case we ever make them.